### PR TITLE
Update localization for DE

### PR DESCRIPTION
**Pull request by lolametro**
Gonna be curious if people understand the German abbreviations for Splat Zones, Rainmaker, ... :)

### CODE REVIEW

File: public/locales/de/common.json [hunk 1]
```diff
@@ -8,10 +8,16 @@
...
+  "pages.maps": "Arenen-Listen",
 
   "header.profile": "Profil",
   "header.logout": "Ausloggen",
   "header.login": "Einloggen",
+  
+  "auth.errors.aborted": "Einloggen abgebrochen",
+  "auth.errors.failed": "Einloggen fehlgeschlagen",
+  "auth.errors.discordPermissions": "Für dein sendou.ink-Profil wird Zugriff auf den Namen, Avatar und Social-Media-Verlinkungen in deinem Discord-Profil benötigt.",
+  "auth.errors.unknown": "Das Einloggen über Discord ist aus einem unbekannten Grund fehlgeschlagen. Falls das weiterhin passiert, kontaktiere bitte für mehr Hilfe.",
```
Comment (remmycat) [inline public/locales/de/common.json:20]: These keys already exist at the bottom of the file, I think I accidentally put them into a different position than the ones in the english json 😅 Feel free to select your translations over mine where you prefer them, I think yours sound a bit more natural.

**Comment by lolametro** [comment]
Oh, that’s unfortunate, I didn’t see that! I also checked the translation progress MD file, but it seems it was not updated with the keys you added already.
I’ll take a look tomorrow and will pick the best of both translations!

Am 21.10.2022 um 22:03 schrieb Remmy Cat Stock ***@***.***>:

﻿

@remmycat commented on this pull request.

________________________________

In public/locales/de/common.json<https://github.com/Sendouc/sendou.ink/pull/1033#discussion_r1002146215>:

> +
+  "auth.errors.aborted": "Einloggen abgebrochen",
+  "auth.errors.failed": "Einloggen fehlgeschlagen",
+  "auth.errors.discordPermissions": "Für dein sendou.ink-Profil wird Zugriff auf den Namen, Avatar und Social-Media-Verlinkungen in deinem Discord-Profil benötigt.",
+  "auth.errors.unknown": "Das Einloggen über Discord ist aus einem unbekannten Grund fehlgeschlagen. Falls das weiterhin passiert, kontaktiere bitte für mehr Hilfe.",


These keys already exist at the bottom of the file, I think I accidentally put them into a different position than the ones in the english json 😅 Feel free to select your translations over mine where you prefer them, I think yours sound a bit more natural.

File: public/locales/de/common.json (hunk 1 above)
Comment (lolametro) [reply public/locales/de/common.json:20]: Oh, that’s unfortunate, I didn’t see that! I also checked the translation progress MD file, but it seems it was not updated with the keys you added already.
I’ll take a look tomorrow and will pick the best of both translations!

Comment (remmycat) [reply public/locales/de/common.json:20]: Huh, not sure what happened there, I could've sworn I ran the script in the end 🤔 Sorry, and thanks!

Comment (lolametro) [reply public/locales/de/common.json]: Done - I hope it's fine now!
Someone now also has to update the MD file so hopefully it doesn't happen again 😄

**Comment by Sendouc** [comment]
🇩🇪 👍

**Pull request merged**